django/contrib/gis/gdal/raster/band.py

```diff
@@ -222,8 +222,10 @@ def data(self, data=None, offset=None, size=None, shape=None, as_memoryview=Fals
             if as_memoryview:
                 return memoryview(data_array)
             elif numpy:
+                # reshape function need the size tuple with the height first
+                reshape_size = (size[1], size[0])
```

Comment:
size.reverse() does not exist on tuple, if you want to get something like this there is two solutions:
```python
size[::-1]
```
or
```python
tuple(reversed(size))
```
What do you think ?